src/shared/ui/subtitle/subtitle.module.scss

```diff
@@ -0,0 +1,32 @@
+.subtitle {
+  margin: 0;
```

Comment:
Maybe we should make margin-top 20px like in the layout. Then we won't need to define it in each Subtitle styles

![image](https://github.com/user-attachments/assets/edf1acba-c9e2-4a6b-a1f0-2c37c8c4bfa1)

Reply:
I accepted all the changes from the comments above, but I’m afraid to change the margin until our teamlead agrees to it...

Reply:
I suppose it will be better to add `margin: 0` to `index.scss`